dev/com.ibm.ws.org.eclipse.jdt.core/bnd.bnd

```diff
@@ -12,11 +12,11 @@
 bVersion=1.0
 
 Bundle-Name: JDT Compiler
-Bundle-SymbolicName: com.ibm.ws.org.eclipse.jdt.core.3.22.0
-Bundle-Description: JDT Compiler, version 3.22.0 [4.16]
+Bundle-SymbolicName: com.ibm.ws.org.eclipse.jdt.core
+Bundle-Description: Eclipse Java Compiler, version 3.26.0: 2020-05-30
```

Comment:
technically this is the 2021-06 simultaneous release: https://wiki.eclipse.org/Category:SimRel-2021-06

Reply:
Ah, good catch. I cherry picked from the 3.22 PR. I'll correct it